backend/server/src/test/java/ai/verta/modeldb/ModeldbTestConfigurationBeans.java

```diff
@@ -132,4 +155,85 @@ private static TestConfig initializeTestConfig() {
     System.getProperties().put("server.port", testConfig.getSpringServer().getPort());
     return testConfig;
   }
+
+  private static class TestAuthServiceChannel extends AuthServiceChannel {
```

Comment:
this was the key to fixing all the breakages...basically removing one layer of mocking, so we're not mock->mock->mock any more